Formula/sdl2.rb

```diff
@@ -21,6 +21,13 @@ class Sdl2 < Formula
       url "https://hg.libsdl.org/SDL/raw-rev/af22dd6c0787"
       sha256 "df68efb43e451789c1bf2873dabc9a70c66264f8b7ad360a71ea4c643c6acc37"
     end
+
+    # Fix make install command.
+    # https://bugzilla.libsdl.org/show_bug.cgi?id=5399
```

Comment:
Could you also add a comment indicating when this patch can be removed? Is it the next version?

That way the next person who updates this formula doesn't need to do a lot of research as to whether they can remove this patch or not.

Reply:
Done.